include/DDML/LoadHdf5.h

```diff
@@ -49,7 +49,8 @@ class LoadHdf5 : public InferenceInterface {
   std::vector<float> m_library{};
 
   // shower library dimensions
-  std::vector<unsigned long> m_dimsOut{};
+  std::vector<unsigned long long> m_dimsOut{}; // This is a hot fix for older HDF5 versions!
+  //std::vector<unsigned long> m_dimsOut{};
```

Comment:
Can we solve this with pre-processor checks? I suppose HDF5 exposes that somehow. Alternatively figure out the minimum version for hdf5 to go with the original version so that we can put this into the dependencies properly.